File: jumble/hello-scons/greeter.cpp

#include "greeter.h"

Greeter::Greeter( const int x ) {
    this->x = x;
}

Greeter::~Greeter() {
}

void
Greeter::PrintSingleGreetings() const {
    std::cout << "Hello, world! GODDAMNIT!" << std::endl;
}

void
Greeter::PrintMultipleGreetings() const {
    for ( int i = 0; i < x; i++ ) {
        std::cout << "Hello, world! GODDAMNIT!" << std::endl;
    }
}

File: jumble/HelloJavaWorld/src/Greeter.java
/**
 * Created by montreal on 11/19/15.
 */

public class Greeter {
    private String greetings;

    Greeter() {
        this.greetings = "Hello, World";
    }

    Greeter( String greetings ) {
        this.greetings = "Hello " + greetings;
    }

    public void ToGreet() {
        System.out.printf( "%s%n", this.greetings );
    }
}

